vendor/github.com/intel/govmm/qemu/qemu.go

```diff
@@ -1864,6 +1864,9 @@ const (
 	// Host is for using the host clock as a reference.
```

Comment:
please create a PR for this change in https://github.com/intel/govmm/  once it's merged, revendor govmm using `dep` command

Reply:
See also:

- https://github.com/kata-containers/community/blob/master/VENDORING.md
- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#re-vendor-prs

Reply:
Will do it after it is merged.